apps/aecore/src/aec_fork_block_settings.erl

```diff
@@ -251,7 +251,14 @@ read_preset_contracts(?LIMA_PROTOCOL_VSN = Release) ->
     end.
 
 accounts_file_name(Release) ->
-    filename:join([dir(Release), accounts_json_file()]).
+    case is_list(os:getenv("AE__SYSTEM__CUSTOM_PREFUNDED_ACCS_FILE")) of
```

Comment:
Unnecessarily complex, it works, but better:
```
case os:getenv("LONG_THING") of
  false -> 
    filename:join([dir(Release), accounts_json_file()]);
  CustomAccsFilePath ->
    lager:info("Custom file for prefunded accounts provided: ~p ~n", [CustomAccsFilePath]),
    CustomAccsFilePath
end
```

Could also possibly check if the file exists, and warn otherwise?!

Reply:
Ah, thanks. 
and yep indeed, I performed the check in the devmode plugin PR but forgot it in here. shipping in a sec.